jhipster-jdl.jdl

```diff
@@ -291,7 +291,6 @@ entity FdaSubmission {
     decisionDate Instant
 
     description TextBlob
-    platform String
```

Comment:
In `MetaImporter`, there's still some outdate code using `setPlatform` on FdaSubmission.

In `fda_submissions.csv` from data folder, the platform can be removed

Reply:
In `FdaSubmissionCriteria`, there's still platform field that can be removed